browser/resources/settings/brave_privacy_page/brave_personalization_options.ts

```diff
@@ -76,13 +83,19 @@ export class SettingsBravePersonalizationOptions extends SettingsBravePersonaliz
   private declare isRequestOTRFeatureEnabled_: boolean
   private declare requestOTRActions_: Object[]
   private declare requestOTRAction_: String
+  private declare isWindowsRecallAvailable_: boolean
 
   browserProxy_: BravePrivacyBrowserProxy = BravePrivacyBrowserProxyImpl.getInstance()
 
   shouldShowRestart_(enabled: boolean) {
     return enabled != this.browserProxy_.wasPushMessagingEnabledAtStartup()
   }
 
+  shouldShowRestartWindowsRecall_(blocked: boolean) {
+    const enabled = !blocked
```

Comment:
same as mentioned below, pick either enabled or blocked, do not use both in different contexts. That is very confusing and error prone